packages/grafana-ui/src/components/Gauge/Gauge.tsx

```diff
@@ -130,9 +137,12 @@ export class Gauge extends PureComponent<Props> {
     }
 
     const decimals = field.decimals === undefined ? 2 : field.decimals!;
+
     if (showThresholdMarkers) {
-      min = +min.toFixed(decimals);
-      max = +max.toFixed(decimals);
+      if (isNumber(min) && isNumber(max)) {
+        min = +min.toFixed(decimals);
+        max = +max.toFixed(decimals);
```

Comment:
Never seen this syntax before. Is it ensuring the value to be positive?

Reply:
It ensures that the value is a numeric value to make sure that the `.toFixed` is available (I guess).